Agda.cabal

```diff
@@ -992,28 +999,23 @@ executable agda-mode
     Agda.Setup
     Agda.Setup.DataFiles
     Agda.Setup.EmacsMode
+    Agda.Setup.LockFile
     Agda.Version
     Agda.VersionCommit
 
   build-depends:
     , base                 >= 4.16.4.0  && < 4.22
     , bytestring           >= 0.11.4.0  && < 0.13
     , directory            >= 1.3.6.2   && < 1.4
-    , filelock             >= 0.1.1.6   && < 0.2
     , filepath             >= 1.4.2.2   && < 1.6
     , gitrev               >= 1.3.1     && < 2
     , process              >= 1.6.16.0  && < 1.7
     , template-haskell     >= 2.18.0.0  && < 2.24
 
--- We use a Cabal `executable` instead of a `test-suite`
---   since we want to avoid re-building Agda when test data changes.
--- Unfortunately, using a `test-suite` breaks this property under nix:
---   * Cabal runs `test-suite`s as soon as they are built
---   * `developPackage` rebuilds Agda when re-building `test-suite`s
---   * https://github.com/NixOS/nixpkgs/issues/9281
--- So instead we build `agda-tests` as an `executable`
---   (alongside Agda, when building with nix),
---   and run it later manually.
```

Comment:
Shouldn't this comment be retained?  It justifies why we do not use a `test-suite` format for `agda-tests`, but an executable.